template-sample-app/template/src/components/layers/StoresLayer.js

```diff
@@ -28,7 +28,7 @@ export default function StoresLayer() {
 
   if (storesLayer && source) {
     return new CartoSQLLayer({
-      id: 'storesPointLayer',
+      id: LayerStyle.id,
```

Comment:
I think id should be extracted from LayerStyle object, as it is conceptually not included inside style props, what do you think?

Reply:
An exported const at the top could be useful, as with sources

Reply:
@VictorVelarde that change is contemplated in the next PR where @Clebal modifies the generator Layer